codalab/lib/bundle_cli.py

```diff
@@ -1699,15 +1702,15 @@ def do_wrm_command(self, args):
             client.delete_worksheet(worksheet_uuid, args.force)
 
     @Commands.command(
-        'wcp',
-        help='Copy the contents from one worksheet to another.',
+        'wadd',
+        help='Append all the items on one worksheet to another. Bundles that do not yet exist on the destination service will be copied over.',
         arguments=(
             Commands.Argument('source_worksheet_spec', help=WORKSHEET_SPEC_FORMAT, completer=WorksheetsCompleter),
             Commands.Argument('dest_worksheet_spec', help=WORKSHEET_SPEC_FORMAT, completer=WorksheetsCompleter),
             Commands.Argument('-c', '--clobber', help='clobber everything on destination worksheet with source (instead of appending)', action='store_true'),
```

Comment:
Explicitly say not deleting the bundles on dest worksheet.
Maybe a better name is '--replace'?